src/EditorFeatures/Core/Implementation/LanguageServer/VisualStudioInProcLanguageServer.cs

```diff
@@ -1,105 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Immutable;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.LanguageServer;
-using Microsoft.CodeAnalysis.Options;
-using Microsoft.CodeAnalysis.Shared.TestHooks;
-using Microsoft.VisualStudio.LanguageServer.Protocol;
-using Roslyn.Utilities;
-using StreamJsonRpc;
-
-using LSP = Microsoft.VisualStudio.LanguageServer.Protocol;
-
-namespace Microsoft.CodeAnalysis.Editor.Implementation.LanguageClient
-{
-    /// <summary>
-    /// Implementation of <see cref="LanguageServerTarget"/> that also supports
-    /// VS LSP extension methods.
-    /// </summary>
-    internal class VisualStudioInProcLanguageServer : LanguageServerTarget
```

Comment:
no longer needed since we don't have LSP push diagnostics and we don't need to specify jsonrpcmethods explicitly anymore 🎉